src/bindings/c/docs/How_to_Create_C_Sample_with_OpenVINO_C_API.md

```diff
@@ -0,0 +1,74 @@
+# How to Create C Sample with OpenVINO C API
```

Comment:
I don't believe that we need to have this guide. I don't believe that someone will add sample for C API

Reply:
Get it. Will remove